service/gcs/runtime/runc/utils_test.go

```diff
@@ -17,11 +18,13 @@ var _ = Describe("Utils", func() {
 	)
 
 	BeforeEach(func() {
-		rtime, err = NewRuntime()
+		var rt runtime.Runtime
+		rt, err = NewRuntime()
```

Comment:
You can probably replace the `=` with a `:=` here, and delete the `var rt runtime.Runtime` line.

Reply:
Fixed.